src/app/complex/complex-results/complex-navigator/table-structure/table-interactor-column/table-interactor-name/table-interactor-name.component.html

```diff
@@ -0,0 +1,15 @@
+<div>
+  <i class="{{interactorOrganismIcon}}"
+     title="{{interactorOrganism}}"></i>
+  {{ interactorName }}
+  <i class="{{interactorTypeIcon}}"
+     title="{{interactorType}}"></i>
+  <!-- Adding of the icons to access details of the interactor -->
+  <a [routerLink]="['/complex/search']"
+     [queryParams]="{query: interactorId, page: 1}"
+     [fragment]="'NAVIGATOR_VIEW'"
```

Comment:
Instead of `NAVIGATOR_VIEW` we need to use `view_complex_navigator`, which is the value `complex-results.component` understand.